inst/resources/activate.R

```diff
@@ -69,6 +69,25 @@ local({
 
   })
 
+  check_project_synchronization <- function() {
```

Comment:
By convention, internal `renv` routines should be prefixed with `renv_` + the scope they're relevant in (except for utility functions that are useful more globally). I would call this `renv_project_synchronized_check()` and put this in `R/project.R`.